src/Microsoft.Build.Tasks.Git.UnitTests/Microsoft.Build.Tasks.Git.UnitTests.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Xunit.Combinatorial" Version="$(XunitCombinatorialVersion)" />
```

Comment:
I'd rather haver this references in the TestUtilities so that it is available to all tests. What's the benefit of moving it here?

Reply:
Having fine grained dependencies make restore more efficient and reduces package dependency graph which is emitted into both `project.assets.json` and `<app>.deps.json`.

That said, I don't have a strong preference here at all. If you want this to remain a transitive dependencies, I'm fine with that as well.

Reply:
I'd prefer simplicity over performance here.

Reply:
Reverted the change